unicodetools/src/main/java/org/unicode/text/UCA/FractionalUCA.java

```diff
@@ -916,7 +936,7 @@ static void writeFractionalUCA(String filename) throws IOException {
             final String gcInfo = getStringTransform(chr, "/", ScriptTransform);
             final String scriptInfo = getStringTransform(chr, "/", GeneralCategoryTransform);
 
-            longLog.print(
+            output.frac.print(
```

Comment:
just checking: Is this intended to be `output.frac.print` and not `output.fracPrint`?

Reply:
It is. Note that the old code wrote to longLog instead of the DualWriter fractionalLog.
This code writes the comment after each mapping, and the comment goes to the regular file but not to the short file.
In the third commit, I move this code into an Output method which writes to frac and fracBlanked but not to fracShort.